web/thesauruses/perl/operators.json

```diff
@@ -0,0 +1,208 @@
+{
+  "meta": {
+    "language": "language_id",
+    "language_version": "version.number",
+    "language_name": "Human-Friendly Language Name"
+  },
+  "categories": {
+    "Arithmetic Operators": [
+      "addition",
+      "addition_assignment",
+      "subtraction",
+      "subtraction_assignment",
+      "multiplication",
+      "multiplication_assignment",
+      "division",
+      "division_assignment",
+      "integer_division",
+      "integer_division_assignment",
+      "modulus",
+      "modulus_assignment",
+      "unary_plus",
+      "unary_minus",
+      "increment",
+      "decrement",
+      "exponential",
+      "factorial",
+      "absolute_value",
+      "percentage"
+    ],
+    "Equality/Comparison Operators": [
+      "equal_to",
+      "not_equal_to",
+      "less_than",
+      "less_than_or_equal_to",
+      "greater_than",
+      "greater_than_or_equal_to",
+      "null_coalescing",
+      "is",
+      "is_not"
+    ],
+    "Logical/Boolean/Bitwise Operators": [
+      "and",
+      "and_assignment",
+      "or",
+      "or_assignment",
+      "not",
+      "not_assignment",
+      "xor",
+      "xor_assignment",
+      "xnor",
+      "xnor_assignment",
+      "left_shift",
+      "left_shift_assignment",
+      "right_shift",
+      "right_shift_assignment"
+    ],
+    "Conditional Operators": [
+      "ternary",
+      "null_forgiving"
+    ]
+  },
+  "operators": {
+    "addition": {
+      "code": "+"
+    },
+    "addition_assignment": {
+      "code": "+="
+    },
+    "subtraction": {
+      "code": "-"
+    },
+    "subtraction_assignment": {
+      "code": "-="
+    },
+    "multiplication": {
+      "code": "*"
+    },
+    "multiplication_assignment": {
+      "code": "*="
+    },
+    "division": {
+      "code": "/"
+    },
+    "division_assignment": {
+      "code": "/="
+    },
+    "integer_division": {
+      "code": "/"
+    },
+    "integer_division_assignment": {
+      "code": "/="
+    },
+    "modulus": {
+      "code": "%"
+    },
+    "modulus_assignment": {
+      "code": "%="
+    },
+    "unary_plus": {
+      "code": "+"
+    },
+    "unary_minus": {
+      "code": "-"
+    },
+    "increment": {
+      "code": "++"
+    },
+    "decrement": {
+      "code": "--"
+    },
+    "exponential": {
+      "code": "**"
+    },
+    "factorial": {
+      "not-implemented": true
+    },
+    "absolute_value": {
+      "not-implemented": true
+    },
+    "percentage": {
+      "not-implemented": true
+    },
+    "equal_to": {
+      "code": "== or eq",
```

Comment:
Could you do something like `==\neq` so it's technically still valid code?

Reply:
Certainly. Updated.